dashboard/lib/api/v1/pd/workshop_score_summarizer.rb

```diff
@@ -80,7 +80,7 @@ def generate_summary_report(workshop: nil, workshops:, course:, facilitator_name
 
     if workshop
       survey_report[:this_workshop] = get_score_for_workshops(
-        workshops: [workshop],
+        workshops: Pd::Workshop.where(id: workshop.id),
```

Comment:
Annoying but necessary. Downstream we expect this to be an activerecord relation of workshops (so we can do joins on it) - this command does the conversion

Reply:
Or it might be more clear if the param was `workshop_id`?